sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.aest.controller.manualReassessment.e2e-spec.ts

```diff
@@ -0,0 +1,236 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeStudentAppeal,
+  createFakeStudentAppealRequest,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import {
+  ApplicationStatus,
+  AssessmentTriggerType,
+  NoteType,
+  StudentAssessmentStatus,
+} from "@sims/sims-db";
+
+describe("AssessmentAESTController(e2e)-manualReassessment", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should create a manual reassessment when requested.", async () => {
+    // Arrange
+    const application = await saveFakeApplication(db.dataSource);
+    application.currentAssessment.studentAssessmentStatus =
+      StudentAssessmentStatus.Completed;
+    application.studentAssessments = [application.currentAssessment];
+    // Add an appeal to the application and current assessment/original assessement
+    const approvedAppealRequest = createFakeStudentAppealRequest();
+    const approvedAppeal = createFakeStudentAppeal({
+      application,
+      studentAssessment: application.currentAssessment,
+      appealRequests: [approvedAppealRequest],
+    });
+    const studentAppeal = await db.studentAppeal.save(approvedAppeal);
+    application.currentAssessment.studentAppeal = studentAppeal;
+    await db.application.save(application);
+
+    const endpoint = `/aest/assessment/application/${application.id}/manual-reassessment`;
+    const payload = { note: "Testing manual reassessment note." };
+    const token = await getAESTToken(AESTGroups.BusinessAdministrators);
+
+    // Act/Assert
+    let responseReassessmentId;
```

Comment:
Very minor. It can be typed to avoid the warning/suggestion from IDE. 

![image](https://github.com/bcgov/SIMS/assets/54600590/b8e6f6bc-980f-46a1-825e-a8e03757b825)